builder/codegen/src/main/java/io/helidon/builder/codegen/GenerateAbstractBuilder.java

```diff
@@ -724,29 +737,88 @@ private static void serviceLoaderPropertyDiscovery(Method.Builder preBuildBuilde
     private static void serviceRegistryProperty(Method.Builder preBuildBuilder,
                                                 PrototypeProperty property) {
         TypeName typeName = property.typeHandler().declaredType();
+
+        // Example: .of("bean-name") or .empty()
+        var namedQualifierFromAnnotation = property.qualifiers()
+                .stream()
+                .filter(a -> a.typeName().equals(TypeName.create("io.helidon.service.registry.Service.Named")))
```

Comment:
Please create a constant for this in `io.helidon.builder.codegen.Types`

Reply:
thx